environment/prod/deployment/beta/Makefile

```
@@ -0,0 +1,39 @@
+export COMPOSE_DOCKER_CLI_BUILD = 1
+export DOCKER_BUILDKIT = 1
+
+DOCKER_COMPOSE_FILENAME = docker-compose.beta.yml
+DOCKER_COMPOSE_APP_CONTAINER = keating-beta-app
+
+DOCKER_EXEC_SCRIPT = docker compose --file ${DOCKER_COMPOSE_FILENAME} exec --workdir /application/environment/prod/deployment/scripts ${DOCKER_COMPOSE_APP_CONTAINER} bash
+DOCKER_EXEC_ARTISAN_COMMAND = docker compose --file ${DOCKER_COMPOSE_FILENAME} exec ${DOCKER_COMPOSE_APP_CONTAINER} php artisan
+
+BETA_DOCKER_IMAGE = ghcr.io/blumilksoftware/keating:beta
+
+CURRENT_DIR = $(shell pwd)
+
+beta-deploy: decrypt-beta-env create-deployment-file
+	@docker compose --file ${DOCKER_COMPOSE_FILENAME} pull && \
+	@docker compose --file ${DOCKER_COMPOSE_FILENAME} up --detach && \
```

```suggestion
	docker compose --file ${DOCKER_COMPOSE_FILENAME} up --detach && \
```

Only first line can contain `@` in multiline command.
```The @ symbol is commonly seen at the beginning of an action lines and means that the action line itself is not be be echoed on the screen as it is executed.```